stable/opsportal/templates/ingress-opsportal-roles.yaml

```diff
@@ -32,7 +32,6 @@ rules:
 - nonResourceURLs:
   - {{ .Values.opsportalRBAC.path | trimSuffix "/"}}
   - {{ .Values.opsportalRBAC.path | trimSuffix "/" }}/*
-  - {{ .Values.opsportalRBAC.graphqlPath | trimSuffix "/"}}
```

Comment:
Don't want the original message to get lost.

Reply:
Yes, it was redundant